build/goenv.txt

```diff
@@ -0,0 +1,7 @@
+GOCACHE=/tmp/go-cache
+GOPROXY=file:///mnt/goproxy
+GOPATH=/home/circleci/.go_workspace:/usr/local/go_workspace
+GOOS=linux
+GOARCH=amd64
+CGO_ENABLED=1
+GO111MODULE=on
```

Comment:
Since these come from the `apro-abi.txt` file, I would prefer to create this file from the Makefile, rather than checking it it. OTOH, the new AES build system isn't y et producing `apro-abi.txt`, and I don't expect another release of apro.

Reply:
`apro-abi@0.10.0.mk` sets the same environment variables so we will use that